scanner/Makefile

```diff
@@ -133,6 +143,14 @@ db-image: copy-db-scripts copy-db-image-scripts copy-db-sigs
 ## Tools ##
 ###########
 
+.PHONY: scanner-go-check-ver
+scanner-go-check-ver:
+ifdef CI
+ifneq ($(GO_VERSION),$(EXPECTED_GO_VERSION))
+	$(error 'Go version "$(GO_VERSION)" does not match the expected "$(EXPECTED_GO_VERSION)" version.')
+endif
+endif
+
```

Comment:
Please let me know why you moved this to the Tools section. To me, it makes more sense to bring this and "deps" to the "Go Build" section or leave it where it was: an additional check on the "binaries we depend on".

Nitpick: Create a var called `build-deps-t := deps` and modify the `deps:` rule accordingly.

Reply:
i'll try and we'll see how that goes. My Makefile skills are lacking

Reply:
feel free to resolve this, if I did what you were imagining

Reply:
LGTM.